docs/UserGuide.md

```diff
@@ -782,15 +795,31 @@ $$
 
 ##### Examples:
 
-TODO
+```
+cca_delete c/NUS Cycling 
+```
+```
+cca_delete c/Jogging
+```
 
 ##### Outcome:
+###### Before delete
+<img src="images/UG-Basics/outcomeCca_deleteBefore.png" width=700>
 
-TODO
+###### After delete
+<img src="images/UG-Basics/outcomeCca_deleteAfter1.png" width=700>
+<img src="images/UG-Basics/outcomeCca_deleteAfter2.png" width=700>
+<box type="tip" seamless>
+
+**Tip:**
+*  After implementing this command, it will list people with relative deleted CCA. So it's normal when you saw
```

Comment:
What does this mean?

Reply:
this means when there are multiple cca exist. after doing this. it lists the people who exist in the delete cca. rest people will be put at the bottom where Jules implements. This contrasts to the list command which shows all the people. So I write a warning here.